reviews/tests.py

```diff
@@ -44,9 +44,19 @@ def setUp(self):
         generate_pdf(self.proposal, 'proposals/proposal_pdf.html')
         self.study = Study.objects.create(proposal=self.proposal, order=1, compensation=Compensation.objects.get(pk=2))
 
+    def check_subject_lines(self, outbox):
+        """
+        Make sure every outgoing email contains a reference number and the
+        text FETC-GW
+        """
+        for message in outbox:
+            subject = message.subject
+            self.assertTrue('FETC-GW' in subject)
```

Comment:
Will this always return the Dutch version? The test passes, but maybe not in the future